### DISCUSSION TITLE
Trouble using ExpressionConfiguration

### DISCUSSION THREAD

**Discussion by sfc-gh-sjames** [ezylang/EvalEx#336] [General]
I am trying to use some of your examples to build an ExpressionConfiguration but getting an error.   Am I missing an import?

import java.math.*;
import com.udojava.evalex.*;
.......
ExpressionConfiguration configuration =
    ExpressionConfiguration.builder()
        .mathContext(new MathContext(32, RoundingMode.HALF_UP))
        .build();

error: cannot find symbol ExpressionConfiguration configuration = ExpressionConfiguration.builder() ^ symbol: class ExpressionConfiguration location: class EvalEx /InlineCode.java:17

**Comment by RobertZenz** [comment]
Yes, `ExpressionConfiguration` lives in the package `com.ezylang.evalex.config`.

Was the example with the wrong imports?
